src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPrintMembers.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// The `bool PrintMembers(StringBuilder)` method is responsible for printing non-derived
```

Comment:
>non-derived [](start = 82, length = 11)

The comment mentions "printing non-derived ... members, and delegating to the base to print inherited ... members". Given that it delegates to the base, this should probably state "members declared in the containing type" rather than "non-derived members".